queries/liquidsoap/highlights.scm

```diff
@@ -0,0 +1,84 @@
+(op) @operator
+["and" "or" "not"] @keyword.operator
+
+[
+  "def"
+  "fun"
+  "let"
+  "def_end"
+] @keyword.function
+
+(let_decoration) @keyword.coroutine
+
+[
+  "open"
+  "begin"
+  "block_end"
+] @keyword
+
+[
+  "while"
+  "do"
+  "to"
+  "for"
+  "for_end"
+  "while_end"
+] @repeat
+
+[
+  "if"
+  "then"
+  "elsif"
+  "else"
+  "if_end"
+] @conditional
+
+[
+  "try"
+  "catch"
+  "try_end"
+] @exception
+
+(inline_if [ "?" ":" ] @conditional.ternary)
+
+[
+  "%ifdef"
+  "%ifndef"
+  "%ifencoder"
+  "%ifnencoder"
+  "%ifversion"
+  "%else"
+  "%endif"
+  "%argsof"
+  "%include"
+] @preproc
+
+(encoder_name) @constant.builtin
+
+(anonymous_argument (var) @parameter)
+(labeled_argument label: (var) @parameter)
+
+"." @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+]  @punctuation.bracket
+
+(app name: (var) @function.call)
+(method) @method
+(method_app) @method.call
+
+(var) @variable
```

Comment:
Should be at the top, precedence is last wins so this will override function calls and parameters.

Reply:
Good to know thanks. Applied.